src/cli/subcommands/state_cmd.rs

```diff
@@ -35,7 +37,12 @@ impl StateCommands {
     pub async fn run(self, api: ApiInfo) -> anyhow::Result<()> {
         match self {
             Self::Fetch { root, save_to_file } => {
-                println!("{}", api.state_fetch_root(root, save_to_file).await?);
+                let ret = api
```

Comment:
I think this should be `StateFetchRoot::call(&client, ...)` - `ApiInfo` should be removed

Reply:
That makes sense. Since this PR has been queued. Let's refactor this in another one. : )